browser/importer/BUILD.gn

```diff
@@ -32,6 +32,7 @@ source_set("importer") {
     # into importer target's deps.
     "//chrome/common:buildflags",
     "//chrome/common/importer",
+    "//chrome/common/importer:interfaces",
```

Comment:
Could you add some comments before this line since it might not be obvious cuz it's for an indirect dep?

Reply:
Done.